get_library.sh

```diff
@@ -129,7 +129,7 @@ elif [ -v HASH ]; then
 elif [ $1 == 'bfl' ]; then
     mv $prefix/bfl-release-release-indigo-bfl-0.7.0-6 $prefix/bfl-0.7.0
 elif [ $1 == 'eigen' ]; then
-    mv $prefix/eigen-eigen-b3f3d4950030 $prefix/eigen-3.3.5
+    mv $prefix/eigen-eigen-* $prefix/eigen-3.3.5
```

Comment:
The destination directory has to match the second parameter of `run_cmd` above (i.e. use `eigen-3.3.5` above, or use `eigen` here).

Reply:
I will change `run_cmd` in [do_everything.sh:349](https://github.com/Intermodalics/ros_android/pull/23/files/7a25d93c9076e8ca60417c5be4f4b771bb076908#diff-82e657dd37ba5e5e07be9d0de6b1dbb6R349) instead. I accidentally removed the suffix in 7a25d93 when cherry-picking this patch from another working branch.